src/Http/Event/ReadEventRestController.php

```diff
@@ -111,7 +111,7 @@ public function getCalendarSummary(string $cdbid, Request $request): string
 
         $style = $request->query->get('style', 'text');
         $langCode = $request->query->get('langCode', 'nl_BE');
-        $hidePastDates = $request->query->get('hidePast', false);
+        $hidePastDates = (bool) $request->query->get('hidePast', false);
```

Comment:
This solution is perfect for me. I just want to point out that the `Request` is a `ParameterBag` and contains "typed" methods like `getBoolean`

Reply:
Thanks I'll keep that in mind! It will get refactored anyway when we switch to PSR-7 request/response objects